text-classifier/SigOpt Text Classifier Walkthrough.ipynb

```diff
@@ -25,18 +25,7 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "POSITIVE: My husband is a BMW mechanic and he drives cars all day.  I wish I could get him one for every car\n",
-      "\n",
-      "NEGATIVE: Far better models for similair price range - dissapointed with the quality of the finish etc etc. works fine though.\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
I originally left the outputs here to make the blog post more readable as it went along, if you think we should remove this you may want to take a look at Scott's Vegas blog post after this one